app/views/index/logs.phtml

```diff
@@ -1,34 +1,62 @@
 <?php /** @var FreshRSS_View $this */ ?>
-<main class="post content">
+<?php $this->partial('aside_configure'); ?>
+<main class="post">
 	<div class="link-back-wrapper">
 		<a class="link-back" href="<?= _url('index', 'index') ?>"><?= _t('gen.action.back_to_rss_feeds') ?></a>
 	</div>
 
 	<h1><?= _t('index.log') ?></h1>
-	<form method="post" action="<?= _url('index', 'logs') ?>"><p>
-		<input type="hidden" name="_csrf" value="<?= FreshRSS_Auth::csrfToken() ?>" />
-		<input type="hidden" name="clearLogs" />
-		<button type="submit" class="btn"><?= _t('index.log.clear') ?></button>
-	</p></form>
+	
 
 	<?php
 		/** @var array<FreshRSS_Log> $items */
 		$items = $this->logsPaginator->items();
 	?>
 
 	<?php if (!empty($items)) { ?>
-	<div class="loglist">
-		<?php $this->logsPaginator->render('logs_pagination.phtml', 'page'); ?>
-
+	<?php $this->logsPaginator->render('logs_pagination.phtml', 'page'); ?>
+	<div id="loglist-wrapper">
+		<table id="loglist">
+			<thead>
+				<th>Log level</th>
+				<th>Date</th>
+				<th>Log Message</th>
```

Comment:
Hard-coded text. Should probably be i18n or removed if we can avoid them.

Reply:
Done